tests/javascript_newsletter_editor/mochaTestHelper.js

```diff
@@ -38,8 +39,9 @@ global._ = _;
 testHelpers.loadScript('tests/javascript_newsletter_editor/testBundles/vendor.js', global.window);
 const Handlebars = global.window.Handlebars;
 global.Handlebars = global.window.Handlebars;
-// Fix global access for Element. It is used in tinymce
+// Fix global access for Element and Node. It is used in tinymce
 global.Element = global.window.Element;
+global.Node = global.window.Node;
```

Comment:
Without this line, I was getting the error below. I'm not sure what are the implications of adding this here.

```
ReferenceError: Node is not defined
    at eval (webpack-internal:///./node_modules/tinymce/tinymce.js:1493:78)
```

Reply:
I assume this happens because tinymce access the Node and Element properties from global space so it works for browser but not in Node JS. I'm not sure about the implications either.